### ISSUE TITLE
Id item

### ISSUE THREAD

**Issue by Worlingkit** [WeakAuras/WeakAuras2#2886]
I can't make a single Item.
Either the blue square is called or the npc of this code.
for example Article number: 46609
https://www.wowhead.com/item=36942/frostmourne
other. (Phantom Sorrow, Fallen Ashbringer ....)
At the same time, you can summon NPCs without any problems. Any, and mounts too.

I also have a WoW Model Viever. Perhaps you can import from there. But on the other hand, he himself takes from the game.

So how do I summon a model? My goal is to use a model from the game, instead of a texture (since it takes me a long time to make it, and the 3D effects are also interesting)
![image](https://user-images.githubusercontent.com/77326609/104299694-f54eff00-54de-11eb-85d1-aaef64237848.png)
![image](https://user-images.githubusercontent.com/77326609/104299731-ff70fd80-54de-11eb-9571-068134ee8d90.png)
![image](https://user-images.githubusercontent.com/77326609/104299749-0566de80-54df-11eb-881d-11cf8e022a3c.png)

**Comment by InfusOnWoW** [comment]
I'm not sure what the wow model viewer displays there as the display id. It doesn't match any id weakauras uses.

The closest you can get is this:
!WA:2!DnvqVnntq0efKkYGQGGeWrksT9JlLMqP9iuNMOeuin1oLpK4qyT3XXRWzx3zxNMKtOCaX58tiN5uK4ghq9ehTQ4os8tG)aH12r0aN8SZ(8SV3mVjFTI9lsls)yP59fuiOH8uot5)thbsb0K4(okkcdVLzaB8ycsFqhHiqXcltczV9ssKYxGhhQycU0ahDSNNeuBpNWD15BlyCLtLQT6u1ctRE3H5NNv5kIab(Icfk0UWU78uK4MwI9TKkcQmmzAsy44P)i9nMOqwVEakxFlCz4hMrbNipVoJcbSE1MTRDAttL(GfjcjLNidHGGguPXpLroWaGRS1OzdN3TYH2D6A35qRoMr6hXjmGmcW)aRncAyw2TR2SPvAgN6ajq5BXj9bPX0i(skyeE7ewpa6KD(LAb(PC)slza5KGxP5RwsNCrMW9ybqdkwAVspP8bhmLWz9jjsE)4Bwl(oMCbhUaisWwHaVNYFTlPAHKGOBISqj4k4u5KemjIoEtZ(egx)VXBgV1AXBh)FXpshU()MjBS2vQ015ntqHkTO5xj9WVCTzA5Bb9sNIo(aRNVQ(Z9xS4hlwS4xzdZAOw)jVSJDfRQvBzgii0AMs2y4Uw9J0EcdtDx39UX7zy5gqKYKihfjq3dtcV0zPz6Vh9tKWzra3fkyjD1Gl4Kzq8NL9TkThy2s3EMHPemHcwPKFjaBnd24k5ejpzQec8snFXx)QlWXRk6X5rcDGwAG)kqg955Ej6upfiksHLM2X5D0nFpwpJzkH7GSb79Z3D6dz0YTGZxcB0k1hhM36Cgv5N2gFMUnU033GRxqUx5igDEPDfhmW3U0)F2rxKDz26ZncVvADoIjtmNn4EcFCy2D1)k(TCVpxUCZsSdSS9U5mnemZpzmlJdHeLF4gPRbYhtqxchcfNdOMCef011hKQD6xUOS8rWGId((R)9p

**Comment by Worlingkit** [comment]
1.your code fails to import.
2. I was able to find models of some swords not in things.
3. Most likely the tab with things has been removed.
![image](https://user-images.githubusercontent.com/77326609/104404102-57594400-5573-11eb-81b2-aebca8326bdf.png)
![image](https://user-images.githubusercontent.com/77326609/104404119-60e2ac00-5573-11eb-838e-c4a65fab118a.png)
As I understand it, the author deliberately removed this part of the code to prevent people from using items.
